chitterApi.js

```diff
@@ -0,0 +1,93 @@
+class ChitterApi {
+  postUserInfo (username, password) {
+    const correctBody = { user: { handle: `${username}`, password: `${password}` } }
+    fetch('https://chitter-backend-api-v2.herokuapp.com/users', {
+      method: 'POST',
```

Comment:
A common way to DRY up the different can be to create a common private method `buildBaseFetchRequest` (for example), which takes a few arguments such as the method, the request path (after the URL's hostname), and the data, and uses `fetch` with these arguments, then the different methods of the class can be refactored a bit to look lighter:

```js
createBaseRequest('POST', '/users', correctBody)
   .then( // etc... )
```